### DISCUSSION TITLE
can someone explain this line to me?

### DISCUSSION THREAD

**Discussion by AutumnFoto** [nss-day-cohort-47/C47-Discussions#71] [Help Me] [answered]
Your Name
Autumn Fotopoulos

Describe your issue

not exactly understanding this line of code : 
<p class="col col-details">Toppings: ${snackObject.toppings.map(topping => 
							topping.topping.name).join (", ")}</p>

- is it saying - map through the toppings, and then get that topping name and join it by using a comma between the names?

**Comment by BryanNilsen** [comment, marked as answer]
`snackObject.toppings.map(topping => topping.topping.name)`
This code is saying map through the toppings and return a new array consisting of ONLY the topping names (not the whole objects)

`.join (", ")`
And then this part is saying, convert that item to a string, **join**ing each item with a comma and a space.

We can do this all in one line with the power of array methods. Pretty awesome, huh?!!
